satpy/tests/reader_tests/test_seviri_l1b_native.py

```diff
@@ -1314,6 +1314,7 @@ def test_read_physical_seviri_nat_file(full_path):
     with warnings.catch_warnings():
         warnings.filterwarnings("ignore", category=UserWarning)
         scene.load(["VIS006"])
+        assert scene["VIS006"].dtype == np.float32
```

Comment:
Lately we've also added a check that the computed values also have the correct `dtype`.
```suggestion
        assert scene["VIS006"].dtype == np.float32
        assert scene["VIS006"].values.dtype == np.float32
```